MVP Module/Source/Feed Presentation/FeedPresenter.swift

```diff
@@ -13,12 +13,20 @@ protocol FeedView {
 	func display(_ viewModel: FeedViewModel)
 }
 
+protocol FeedErrorView {
+	typealias ErrorMessage = String
+
+	func display(_ errorMessage: ErrorMessage)
+}
```

Comment:
The `message` could be an optional `String?`, so the Presenter can decide when to show/hide the message.

For example, in the `didStartLoadingFeed()` method, the Presenter can call  `feedErrorView.display(.none)` to hide the error.

This way, the ViewController doesn't need to decide when to hide the error. It's easier to manage presentation state transitions when all changes are centralized in the presenter.

Also It can be a bit more convenient to have a `FeedErrorViewModel` struct to describe error view instead of a `typealias ErrorMessage = String`. It is more typesafe and it would be easier to add new parameter (say some error icon) to `FeedErrorView` in future as it would require only the change in `FeedErrorViewModel` and will not require change in `FeedErrorView` protocol.

```suggestion
struct FeedErrorViewModel {
	let message: String?
}

protocol FeedErrorView {
	func display(_ viewModel: FeedErrorViewModel)
}
```